docs/components/segmented-control.md

```diff
@@ -236,24 +236,24 @@ An Example of changing the KSegmentedControl to a purple theme instead of blue m
 
 <style>
 .purple-segment {
-  --KSegmentedControlText: var(--purple-400, #{$kui-method-color-text-connect});
-  --KSegmentedControlSelectedBackground: var(--white, #{$kui-color-background});
-  --KSegmentedControlSelectedBorder: var(--purple-300, #9396fc);
-  --KSegmentedControlUnselectedBackground: var(--blue-100, #{$kui-color-background-primary-weakest});
-  --KSegmentedControlUnselectedBorder: var(--purple-200, #bec0fd);
-  --KSegmentedControlGap: var(--spacing-sm, #{$kui-space-50});
+  --KSegmentedControlText: #473cfb;
+  --KSegmentedControlSelectedBackground: white;
```

Comment:
suggestion: use background color var

Reply:
This is docs file, it's fine to hardcode some colors here.